### ISSUE TITLE
Url wrong when get order from wp-admin edit page

### ISSUE THREAD

**Issue by walfud** [dfactoryplugins/post-views-counter#9]
![default](https://cloud.githubusercontent.com/assets/9317897/17083838/df6d6558-51de-11e6-8bae-e2f74f270e7a.png)
I deploy wordpress in docker which network is internal with 192.168.100.10. but the plugin should not assume that outter network can access it.
The right way is to read the Settings from wordpress to get the public url as below:
![21](https://cloud.githubusercontent.com/assets/9317897/17083834/c10dc8d2-51de-11e6-8108-369dc66bfa5f.png)

I'm so sorry that I can't fix it for you.

**Comment by bartoszarendt** [comment]
Can you describe when the issue occurs? Is it while navigating WP admin - sorting posts by views count?

**Comment by walfud** [comment]
Yes, I'm in admin page, I click the icon want to sort posts by views count. 
I install the plugin by upload file. The plugin version is 1.2.2
